src/packages/excalidraw/CHANGELOG.MD

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## 0.1.0
+
+First release of `@excalidraw/excalidraw`
```

Comment:
Is that going to be manually updated?

Reply:
Not really we can automate this with a script that just fetches the commits after the last release and adds it here.